content/en/security/application_security/software_composition_analysis/setup/compatibility/go.md

```diff
@@ -5,6 +5,9 @@ type: multi-code-lang
 code_lang_weight: 20
 ---
 
+<div class="alert alert-info">This documentation is for the Go Tracer v1.x. If you are looking for v2.x preview documentation, see the <a href="/security/application_security/software_composition_analysis/setup/compatibility/go-v2">Go Compatibility Requirements</a> documentation.</div>
```

Comment:
Similar to DBM, let's have this as a tab for v1 or v2 instead of a separate page